src/index.html

```diff
@@ -10,8 +10,175 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+            Surname:
+          <input
+            type="text"
+            autocomplete="off"
+            name="surname"
+            required
+            >
```

Comment:
minor - closing tag bracket should be at same level as opening
```suggestion
          >
```